pkg/cmd/get/get_build_logs.go

```diff
@@ -430,7 +430,7 @@ func (o *GetBuildLogsOptions) getTektonLogs(kubeClient kubernetes.Interface, tek
 		return true, errors.New("there are no build logs for the supplied filters")
 	}
 
-	if pa.Spec.BuildLogsURL != "" {
+	if pa.Spec.BuildLogsURL != "" && !strings.HasPrefix(pa.Spec.BuildLogsURL, "http") {
```

Comment:
Looking at `logs.StreamPipelinePersistentLogs` I am wondering why we cannot just get the content of the URL in the case of http/https? Are these URLs not pointing to actual logs? My understanding was that if no bucket is provided, logs can be stashed into a GitHub repo. At least for some time I seem to recall looking at logs in a GitHub repo. Assuming this are the kind of HTTP URLs we are talking about here, why don't we just read their content?

Reply:
Here is what I mean https://github.com/jenkins-x/environment-tekton-mole-dev/tree/gh-pages/jenkins-x/logs.

Reply:
As I mentioned offline, https://github.com/jenkins-x/jx/blame/0548010f5597c7e92bbdb6196454878d894cb38f/pkg/logs/tekton_logging.go#L278-L281 is relevant here - @dgozalo can speak more to this.

Reply:
My understanding is that we were moving the storage of persistent logs to buckets at all times, meaning we wouldn't be supporting the git storage anymore. The case in production right now is that the storage configuration gets wiped and it goes back to git storage, but this shouldn't really happen.

The thing is, if buckets storage is not enabled, there shouldn't be any logs sent to GitHub.

Maybe we need to enable getting the logs from GH Pages as a fallback?

Reply:
> Maybe we need to enable getting the logs from GH Pages as a fallback?

+1